tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+    content: [
+        "./resources/**/*.blade.php",
+        "./resources/**/*.js",
+        "./resources/**/*.vue",
```

Comment:
```suggestion
        "./resources/**/*.{js,vue,blade.php}",
```
I'm pretty sure you can do this.